### PR TITLE
add gsplat viewer, intro post, short video

### PR DESCRIPTION
**Pull request by dylanebert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,17 +206,20 @@ GPU, much faster than previous methods, while the generated instances can be dir
 - [WebGL Viewer 2](https://github.com/cvlab-epfl/gaussian-splatting-web)
 - [Three.js](https://github.com/mkkellogg/GaussianSplats3D)
 - [A-Frame](https://github.com/quadjr/aframe-gaussian-splatting)
+- [gsplat.tech](https://gsplat.tech/)
```
Comment (MrNeRF) [inline README.md:209]: I know this resource but I think there is no code available? At least I did not find any or maybe I overlooked it.

**Review by MrNeRF — CHANGES_REQUESTED** [review]
Hi Dylan, 
thank you for your contribution. Just one small remark. I would like to add only open source resources that are educational. I am not aware that gsplat.tech comes with code (maybe I missed it? If it provides code, can you please add a link to the repo? Otherwise I would like to remove it for the aforementioned reasons since it only showcases trained scenes.

File: README.md (hunk 1 above)
Comment (dylanebert) [reply README.md:209]: It's not open source, feel free to remove gsplat.tech if you only want to reference open source viewers :)

Comment (dylanebert) [reply README.md]: I've rebased and removed gsplat, feel free to merge if you're happy with it

**Comment by MrNeRF** [comment]
Cool! Thank you!

**Pull request merged**